### PR TITLE
fix new sonar issues

### PR DESCRIPTION
**Pull request by Gr33ndev**
Part of Gamify-IT/issues#498

### CODE REVIEW

File: tests/unit/GamePanel.spec.ts [hunk 1]
```diff
@@ -1,81 +1,67 @@
...
-      await wrapper.vm.$nextTick();
-      jest.advanceTimersByTime(1000);
-      await wrapper.vm.$nextTick();
-    }
-    await wrapper.vm.$nextTick();
-    expect(cards.length).toBe(0);
-    expect(wrapper.findAllComponents(PairItem).length).toBe(cardLength/2);
-  });
-});
+import { mount, VueWrapper } from "@vue/test-utils";
```
Comment (delvh) [inline tests/unit/GamePanel.spec.ts:1]: Did you perhaps change the line encoding from LF to CRLF or vice-versa in this file?

Comment (Gr33ndev) [reply tests/unit/GamePanel.spec.ts:1]: hopefully not (?)

File: TODO.md
Comment (C0okiedestroyer) [inline TODO.md:1]: We can remove this file completely as it is no longer needed. It was created at the start of this project and then fell into oblivion. Therefore it was not updated and is deprecated.

Comment (Gr33ndev) [reply TODO.md:1]: Which file?

Comment (C0okiedestroyer) [reply TODO.md:1]: TODO.md

**Review by C0okiedestroyer — APPROVED** [review]

**Pull request merged**